examples/i18n/src/resource/german.rs

```diff
@@ -0,0 +1,44 @@
+pub static S: &'static str = r#"
```

Comment:
1. Remove `pub static S: &'static str = r#"` and `"#`
1. Rename the file to `german.ftl`
1. Perhaps create a new folder `resources` or `translations` in the root or in `i18n` folder and move this file into it.
1. Replace `(german::S).to_string()` in `resource.rs` with `include_str!("path_to_the_file").to_string()`

Reply:
Wow. I didn't expect this to be so easy. Works like a charm.